axelrod/load_data_.py

```diff
@@ -0,0 +1,45 @@
+import pkg_resources
```

Comment:
Maybe type annotations are a good check here so when start annotating we'll get a little extra coverage.